pkgs/applications/networking/cluster/rke2/default.nix

```
@@ -5,15 +5,19 @@ let
   extraArgs = removeAttrs args [ "callPackage" ];
 in
 rec {
-  rke2_1_31 = common (
-    (import ./1_31/versions.nix)
-    // {
-      updateScript = [
-        ./update-script.sh
-        "31"
-      ];
-    }
-  ) extraArgs;
+  rke2_1_31 =
+    (common (
+      (import ./1_31/versions.nix)
+      // {
+        updateScript = [
+          ./update-script.sh
+          "31"
+        ];
+      }
+    ) extraArgs).overrideAttrs
+      {
+        meta.knownVulnerabilities = [ "rke2_1_31 has reached end-of-life" ];
```

```suggestion
        meta.knownVulnerabilities = [ "rke2_1_31 has reached end-of-life on 2025-11-11" ];
```
I think we should still include the date as with 1.30 and earlier, how long ago a release went EOL is pretty useful for determining how likely it is to be compromised